logging/src/test/resources/default-subsystem.xml

```diff
@@ -38,9 +38,6 @@
     <logger category="com.arjuna">
         <level name="WARN"/>
     </logger>
-    <logger category="org.apache.tomcat.util.modeler">
-        <level name="WARN"/>
-    </logger>
```

Comment:
Yeah I think it's okay. Like you said the main issue would be with older slaves, but they'd likely need some other changes too.